packages/ts/lit-form/src/StringConverter.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * A converter for a string representation of a value of type T.
+ */
+export type StringConverter<T> = {
```

Comment:
```suggestion
export interface StringConverter<T> {
```
`interface` suits better here logically